src/App.css

```diff
@@ -0,0 +1,40 @@
+.App {
+  display: flex;
+  height: inherit;
+  justify-content: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
```

Comment:
Guessing this (and some other classes in this file) are leftover from create-react-app scaffolding